sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -697,7 +698,8 @@ def _insert_all_rows(
           table_ref_str,
           json_rows=rows,
           row_ids=insert_ids,
-          skip_invalid_rows=True,
+          skip_invalid_rows=skip_invalid_rows,
+          ignore_unknown_values=ignore_unknown_values,
```

Comment:
Just to make sure I'm not missing anything, this is the main change in functionality right? All the other changes are just passing parameters through function signatures until they get here, right?

Reply:
correct!